### PR TITLE
Feature/fit heading truncate

### LINKED ISSUE #655: [Enhancement] Truncate page-title if title is longer then your defined max-height of page-title
**Is your enhancement request related to a problem? Please describe.**
We would like to truncate our page-title when we have longer titles that break our truncation rule (scale from h1 to h3). 

In the page-template we want to define what our max-height of our page-titles are, and we want to scale font down before we truncate.

The text-scaling task can be found here: https://github.com/kirbydesign/designsystem/issues/481

So the rule should be triggered when there is no more space in the title and the font-size has gone down to h3.

**See also**

Left to right: 
H1 - H2 - H3 & truncate
Skærmbillede 2019-10-07 kl  19 50 07

### LINKED ISSUE #871: [BUG] toolbar doesn't truncate on longer texts
**Describe the bug**
toolbar text doesn't truncate on longer texts

Skærmbillede 2020-01-17 kl  13 16 16

**Kirby version**

- v0.8.75

### PR DESCRIPTION
**Pull request by hafstad**
closes #655, closes #871

### CODE REVIEW

File: src/app/examples/page-example/advanced/page-custom-title-example.component.ts [hunk 1]
```diff
@@ -9,20 +9,18 @@ import { BasePageExampleComponent } from '../base-page-example.component';
...
   <ng-template kirbyPageToolbarTitle>
     <ng-container *ngTemplateOutlet="customTitle"></ng-container>
   </ng-template>
 
   <ng-template #customTitle>
     <div style="display: flex;">
-      <div style="overflow: hidden; text-overflow: ellipsis;">
-        Custom Title With a very long name
-      </div>
+      <h1 style="overflow: hidden; text-overflow: ellipsis;" [kirbyFitHeading]="{ maxLines: 2, truncate: true }">Custom Title With a very very very very very very very very very very very very long name</h1>
```
Comment (jonasjensen-brolstark) [inline src/app/examples/page-example/advanced/page-custom-title-example.component.ts]: is the `style` attribute needed here or is it just something you forgot to remove?

**Comment by jkaltoft** [comment]
@hafstad This PR is more than one year old.  It would be great if we could close it 😊. Do you know what the status is?

**Comment by jkaltoft** [comment]
> @hafstad This PR is more than one year old. It would be great if we could close it 😊. Do you know what the status is?

@hafstad

**Comment by hafstad** [comment]
Due the age of this pull-req, I will close this and create a new one to fix the issue

**Pull request closed without merging**